System/GraphicalPrimitive.cpp

```diff
@@ -31,13 +31,43 @@ namespace RTE {
 		drawRightPos.m_Y -= targetPos.m_Y;
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	static void lineWithThickness(BITMAP *drawScreen, const Vector &startPos, const Vector &endPos, unsigned char color, const int &thickness) {
+		int x_delta = startPos.GetFloorIntX() - endPos.GetFloorIntX(),
+			y_delta = startPos.GetFloorIntY() - endPos.GetFloorIntY();
+		Vector n(-y_delta, x_delta);
+
+		int startIdx = 0;
+		if (thickness % 2) { // If odd draw the center line first
+			line(drawScreen, startPos.GetFloorIntX(), startPos.GetFloorIntY(), endPos.GetFloorIntX(), endPos.GetFloorIntY(), color);
+			startIdx = 1;
+		}
+		int i = startIdx;
+		for (; i < thickness / 2; ++i) {
+			float n_len_factor = i / n.GetMagnitude();
```

Comment:
We use camelCase for local variables, not snake_case. Please change this accordingly.
Also, if you can come up with more descriptive names than `n_len_factor` and `n_unit`, please do! Especially given n_unit isn't actually a unit vector (unless I'm horribly mistaken) like the name suggests.